Modules/Messages/src/Messages.Domain/Entities/Message.cs

```diff
@@ -25,7 +25,7 @@ public Message(IdentityAddress createdBy, DeviceId createdByDevice, DateTime? do
 
     public MessageId Id { get; }
 
-    public DateTime CreatedAt { get; internal set; }
+    public DateTime CreatedAt { get; set; }
```

Comment:
By removing the `internal` you just made it worse. Remove the whole setter.

Reply:
I was sleeping, for sure. My bad